src/main.ts

```diff
@@ -1,8 +1,26 @@
 import { NestFactory } from '@nestjs/core';
-import { AppModule } from './app.module';
-
+import { AppModule } from './app/app.module';
+import { DataSource as Connection } from 'typeorm';
+import { testDatabaseConnection } from '@app/common';
+import { Logger, ValidationPipe } from '@nestjs/common';
 async function bootstrap() {
+  const logger = new Logger('bootstrap');
   const app = await NestFactory.create(AppModule);
+  const isDatabaseConnected = await testDatabaseConnection(app.get(Connection));
+
+  app.enableCors({
+    origin: 'localhost:3000',
```

Comment:
make it as a constant in  `utils/constants.ts` file , for example:
```suggestion
    const ORIGIN_URL = 'localhost:3000'
```

Reply:
define all constants in  `utils/constants.ts`